### DISCUSSION TITLE
Returning raw json

### DISCUSSION THREAD

**Discussion by davidmoshal** [ravendb/ravendb#12464] [General]
Anyone know a way to return a raw JSON document?

ie: instead of fetching the document, de-serializing to some class, then serializing back to json, is there a way to just load the document as a json string?

ie:
```kt
val json:String = session.loadRaw("collection", "id")  
// or even:
val json:String = session.loadRaw(Class, "id") // class just used to find the collection, but not for serialization
```

**Comment by ayende** [comment]
What language / environment are you using?
In general, for C#, you can use:

```
         var ids = new List<string> { "employees/1-A", "employees/4-A", "employees/7-A" };
         session.Advanced.LoadIntoStream(ids, stream);
```

The idea here is that you'll typically bypass everything and send this directly to the outgoing network stream.
